examples/scripts/sft_gemma3.py

```diff
@@ -0,0 +1,67 @@
+# Copyright 2025 The HuggingFace Team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import torch
+from datasets import load_dataset
+from transformers import AutoModelForImageTextToText, AutoProcessor
+
+from trl import SFTConfig, SFTTrainer
+from trl.trainer.utils import pad
+
+
+if __name__ == "__main__":
+    # Load dataset
+    train_dataset = load_dataset("open-r1/codeforces-cots", split="train[:100]")
+    train_dataset = train_dataset.remove_columns("prompt")
+
+    # Load model and processor
+    model_id = "google/gemma-3-4b-it"
+    processor = AutoProcessor.from_pretrained(model_id)
+    model = AutoModelForImageTextToText.from_pretrained(model_id, attn_implementation="eager")
```

Comment:
```suggestion
    model = AutoModelForImageTextToText.from_pretrained(model_id, attn_implementation="sdpa")
```

Reply:
@kashif I get this warning:

```
It is strongly recommended to train Gemma3 models with the `eager` attention implementation instead of `sdpa`. Use `eager` with `AutoModelForCausalLM.from_pretrained('<path-to-checkpoint>', attn_implementation='eager')`.
```

Reply:
sdpa gives me OOM currently. Let's keep eager until sdpa is fixed?

Reply:
ok...